main/webapp/modules/core/styles/project/sidebar.less

```diff
@@ -133,6 +150,10 @@ div#body.hide-left-panel a#show-left-panel-button {
   display: inline-block;
 }
 
+.hideFacetSplash {
```

Comment:
Shouldn't be another for the hideHistorySplash ?

Reply:
I didn't think it'll be necessary as the aim of that style is to set the splash display to none. I thought it'll be unnecessary repeition

Reply:
But don't you have 2 splashes? One for history and one for Facet? So, your intention are that when one user closes one, he closes the other also?! This is not clear to me yet.

Reply:
> So, your intention are that when one user closes one, he closes the other also?

Actually yes. Please correct me if my concept is wrong but I was thinking the user could want to close one splash but not the other. So I created two different preferences for the splashes. The `hideFacetSplash` class is reused in those different instances

Reply:
> Actually yes. Please correct me if my concept is wrong but I was thinking the user could want to close one splash but not the other.

K. So, then, this `.hideFacetSplash` could be called `.hideSplash`, `.hideLeftPanelSplash`, `.hideFacetHistorySplash`, but not just Facet, since it will be used in the 2.

Reply:
and maybe just `.hide-splash`, to keep a consistent style across class names?